src/serverlist.cpp

```diff
@@ -87,10 +88,10 @@ CServerListManager::CServerListManager ( const quint16  iNPortNum,
     if ( iServInfoNumSplitItems >= 3 )
     {
         // [this server name]
-        ThisServerListEntry.strName = slServInfoSeparateParams[0].left ( MAX_LEN_SERVER_NAME );
+        ThisServerListEntry.strName = slServInfoSeparateParams[0].remove ( reIllegal ).left ( MAX_LEN_SERVER_NAME );
```

Comment:
Just removing \n automatically without warning the user might cause confusion. I'd give a warning on stderr and strip the new line afterwards.

Reply:
It can go on the Wiki.  It's quite difficult to get a new line into the string in the first place - I doubt people are going to use it.

Reply:
Hmm yes that’s probably more common. Can you open an issue on the website repo to document this change?

Reply:
Done: https://github.com/jamulussoftware/jamuluswebsite/issues/514

Reply:
Thanks!